aten/src/ATen/core/UndefinedTensorImpl.cpp

```diff
@@ -5,7 +5,7 @@ namespace at {
 
 // should this use the globalContext?  Can it get a context passed in somehow?
 UndefinedTensorImpl::UndefinedTensorImpl()
-: TensorImpl(UndefinedTensorId(), ScalarType::Undefined, nullptr, /* is variable */ false) {
+: TensorImpl(UndefinedTensorId(), ScalarType::Undefined, nullptr, TensorImplOptions(false)) {
```

Comment:
I don't think the TensorImplOptions specification is correct: UndefinedTensors don't have strides and don't have storages.